zboxcore/client/entity.go

```diff
@@ -8,13 +8,15 @@ import (
 
 type Client struct {
 	*zcncrypto.Wallet
-	signatureSchemeString string
+	signatureSchemeString string `json:"signature_scheme"`
 }
 
 var client *Client
 
 func init() {
-	client = &Client{}
+	client = &Client{
```

Comment:
@Rizary What's the reason of changing this?

Reply:
@cnlangzi has it fixed in master. Basically because it will caused null pointer when we call wallet.